### PR TITLE
Fix bug in nll calculation.

### PR DESCRIPTION
**Pull request by jlmaccal**

### CODE REVIEW

**Comment by ardizzone** [comment]
Hi!

firstly, thank you for contributing!
However, I am afraid I have to disappoint you, the old loss was correct:
- The 1/ndim is included in the mean of `torch.mean(z**2)`
- In computing the log jacobian, all dimensions are summed, that's why only the jacobian has to be divided.
